Wiki/zh-Tw.js

```diff
@@ -0,0 +1,678 @@
+    PERMISSIONS_NEEDED: "你必須是伺服器 **擁有者** 或是擁有這些權限： {{permission_1}} 或 {{permission_2}} 才能將 Filo 邀請至伺服器",
+    SLASH_COMMAND_WARNING: "目前 Filo 將不支援沒有把機器人斜線指令權限開啟的伺服器，或是註冊 Filo 斜線指令但把 Filo 踢出的伺服器，如此一來你將會收到一個一般錯誤",
+    STEP_1: {
+      TITLE: "打開瀏覽器中的邀請連結",
+      DESCRIPTION: "**[點擊這裡]({{link}})** 我們將會為您導向到邀請頁面和斜線指令權限頁",
+      EXAMPLE: "邀請頁面",
+      ALTERNATIVE_INVITE: "如果你不需要斜線指令的話請點擊 **[這裡的邀請連結]({{link}})**"
+    },
+    STEP_2: {
+      TITLE: "選擇 Filo 要落腳的伺服器",
+      DESCRIPTION: "打開 **新增至伺服器** 的下拉式選單並選擇要添加 Filo 的伺服器",
+      EXAMPLE: "添加伺服器的選擇框 (標註部分)",
+      BEFORE_THE_RESULT_EXAMPLE: "當你選擇好之後，你應該會看到下圖的畫面",
+      CONTINUE: "當完成這個步驟後，你應該會看到 **繼續** 按鈕。不用猶豫了，點下去吧！"
+    },
+    STEP_3: {
+      TITLE: "選擇權限 (可選可不選)",
+      DESCRIPTION: "這個步驟可以讓您選擇 Filo 的伺服器權限 (非頻道權限). 如果沒有特別要求，請點擊 **繼續** 按鈕",
+      INTEGRATION_ROLE: "如果您全選了所有權限，Discord 會幫您建立一個 **Filo**的身分組。反之，Discord 會把權限單獨寫在他的身分裡",
+      PERMISSIONS_NEEDED: "Filo 必須有這些指令才能正常工作: {{permission_1}}、 {{permission_2}}、{{permission_3}} 和 {{permission_4}}"
+    },
+    STEP_4: {
+      TITLE: "授權邀請",
+      DESCRIPTION: "當你選好了權限，點擊 **授權** 按鈕來把 Filo 邀進伺服器"
+    }
+  }
+};
+
+exports.MODULES = {
+  ABOUT_THE_MODULE: "關於本模組",
+  COMMUNITY_RECOMMENDATION: "我們強烈建議你使用本模組來管理社群",
+  TO_ACTION_COMMAND: "要 {{action}} 本模組，請輸入本指令： {{command}}",
+  MAX_AND_MIN_DURATION: "指令有效的持續時間必須在 **{{min}}** 到 **{{max}}** 之間 ",
+  VALUE_GREATER_THAN: "數值必須大於 **{{number}}**",
+  SUMMARY: "概要",
+  IF_YOU_WANT_SETUP: "如果要 {{action}} 本模組，請**[點擊這裡]({{link}})**。我們將會帶你到另一個主題",
+  VARIABLE_WILL_RETURN: "這個變數會傳回 {{data}}",
+  GETTING_STARTED: {
+    DESCRIPTION: "在繼續之前，您必須考慮一系列可能影響您執行的任何操作因素",
+    REQUIRES_ADVANCED_PERMISSION: "Filo 需要要此項進階權限： {{permission}}",
+    REQUIRES_BASIC_PERMISSION: "Filo 需要要此項基礎權限: {{permission}}",
+    REQUIRES_ADVANCED_PERMISSIONS: "Filo 需要以下進階權限: {{permissions}}",
+    REQUIRES_BASIC_PERMISSIONS: "Filo 需要以下基礎權限: {{permissions}}",
+    REQUIRES_USER_PERMISSION: "在執行本主題的指令前，你需要此項權限： {{permission}} ",
+    REQUIRES_USER_PERMISSIONS: "在執行本主題的指令前，你需要以下權限： {{permission}}",
+    CUSTOM_PREFIX: "你需要取代 " + "<kbd>" + "f!" + "</kbd>" + " 成你設定的 Prefix 。如果你需要更換它的教學，請**[點擊這裡]({{link}})**"
+  },
+  MODULE_EXCEPTIONS: {
+    TITLE: "常見錯誤",
+    DESCRIPTION: "如果在使用本模組時，如果你沒有開啟下列敘述，你將會得到錯誤",
+    FILO_REQUIRED_PERMISSION: "如果 Filo 沒有此項權限： {{permission}}",
+    FILO_REQUIRED_PERMISSIONS: "如果 Filo 沒有以下權限： {{permissions}}",
+    USER_HAS_PERMISSION: "如果使用者沒有這個權限： {{permission}}",
+    USER_HAS_PERMISSIONS: "如果使用者沒有以下權限： {{permissions}}",
+    FILO_LOWER_ROLES: "如果 Filo 的身分組位階低於這個使用者",
+    FILO_EQUAL_ROLES: "如果 Filo 的身分組位階等於這個使用者",
+    WEBHOOK_OTHER_MODULE: "如果 Webhook 已經跟其它模組共用 (e.g. {{module}})",
+    SETTINGS_WILL_RESET: "如果要滿足此要求，模組設置將被重置",
+    MODULE_WILL_NOT_WORK: "如果滿足此要求或以下要求，則該模組將無法工作"
+  },
+  ENABLE_MODULE: {
+    TITLE: "如何啟用本模組？",
+    SUBTITLE: "啟用模組",
+    SUPPLEMENTARY_MODULE: "這個模組可以補充 **[{{module}}]({{link}})**"
+  },
+  DISABLE_MODULE: {
+    TITLE: "如何停用本模組？",
```

Comment:
Why is there a question here?
The model is "How to disable this module" and with all the translations it is "Steps to disable this module".